engine/flink/model-util/src/test/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/TransformersTest.scala

```diff
@@ -197,19 +260,18 @@ class TransformersTest extends FunSuite with Matchers {
   }
 
   private def validateConfig(aggregator: String, aggregateBy: String): CompilationResult[Unit] = {
-    validator.validate(process("aggregate-sliding", aggregator, aggregateBy))
+    validator.validate(process("aggregate-sliding", aggregator, aggregateBy, emitWhenEventLeft = Some(false), emitExtraWindowWhenNoData = None))
   }
 
-  private def process(aggregatingNode: String, aggregator: String, aggregateBy: String) = {
+  private def process(aggregatingNode: String, aggregator: String, aggregateBy: String, emitWhenEventLeft: Option[Boolean], emitExtraWindowWhenNoData: Option[Boolean]) = {
```

Comment:
I wonder why there is Option[Boolean]? What is difference between none and false in this case?

Reply:
Because we have one process for both sliding and tumbling and both of them has two different parameters. Other option would be to have two processWithSliding and processWithTumbling methods but after all both of them would invoke such a method or have a lot of code duplication